integration/kubernetes/confidential/snp.bats

```diff
@@ -14,6 +14,9 @@ UNENCRYPTED_IMAGE_URL="${IMAGE_REPO}:unencrypted"
 # Text to grep for active feature in guest dmesg output
 SNP_DMESG_GREP_TEXT="Memory Encryption Features active:.*SEV-SNP"
 
+# Add sleep to give nydus snapshotter a chance to start-up as suggested by AMD folks
```

Comment:
I'm not sure how bats executes commands outside of the main tests, setup or teardown methods. The snp test is still failing. I would recommend moving this to the `setup_file` method. However, I think this check should go in the `containerd_nydus_setup.sh`. Maybe SEV/SNP tests are the only things experiencing issues with this right now, but that doesn't necessarily mean that other things won't have issue with the snapshotter not being fully initialized later.